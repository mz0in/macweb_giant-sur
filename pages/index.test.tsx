import React from 'react'
import { shallow, ShallowWrapper } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import App from './index'

describe('Home page', () => {
  let app: ShallowWrapper

  beforeEach(() => {
    app = shallow(<App />)
  })

  it('Should renders properly', () => {
    expect(shallowToJson(app)).toMatchSnapshot()
  })
})
