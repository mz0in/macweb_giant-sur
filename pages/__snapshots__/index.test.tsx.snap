// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page Should renders properly 1`] = `
<div
  className="flex mx-auto w-screen h-screen bg-wallpaper bg-cover overflow-hidden"
>
  <MenuBar />
  <Desktop />
  <Dock />
</div>
`;
